### PR TITLE
Fix similar code checking in codeclimate

### PR DESCRIPTION
**Pull request by LiKang6688**
similar-code threshold describes the mass code block to be analyzed for duplication.
Refer https://github.com/codeclimate/codeclimate-duplication
for this settings.

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -1,4 +1,11 @@
 version: "2"
+plugins:
```
Comment (haxxmaxx) [inline .codeclimate.yml:2]: Does this need to be under `plugin`? Does it work putting it under `checks`, as the other settings?

Comment (LiKang6688) [reply .codeclimate.yml:2]: Following the Configuring in https://github.com/codeclimate/codeclimate-duplication, it needs to be under plugin. We can use Code Climate CLI (https://github.com/codeclimate/codeclimate) to do analyze locally to see if it works as we want.

**Comment by haxxmaxx** [comment]
I just tried this rule on this branch https://github.com/qlik-oss/sn-table/pull/109. With the threshold at 3 it does not complain about `styling-utils`, which I think is a bit too weak. Those two functions could be replaced with one function taking one extra argument, which is something I think code climate should catch, but then you also get the false positives, like the propTypes.

File: .codeclimate.yml [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
+  duplication:
+    enabled: true
+    config:
+      languages:
+        filters:
+          - "(object (Identifier PropTypes))"
```
Comment (LiKang6688) [inline .codeclimate.yml:8]: Based on https://github.com/codeclimate/codeclimate-duplication/issues/89

Comment (haxxmaxx) [reply .codeclimate.yml:8]: Would be really nice if this works, but I still see 6 fixed issues on codeclimate, there should only be the two propTypes that are fixed

Comment (LiKang6688) [reply .codeclimate.yml:8]: okay, look into it

Comment (LiKang6688) [reply .codeclimate.yml]: now it is correct

Comment (haxxmaxx) [reply .codeclimate.yml]: I don't know why, but if you look at codeclimate it is saying that the propTypes duplications are fixed, then there are two new issues that are exactly the same... We can try to merge this and see what it looks like when ir runs on main

Comment (LiKang6688) [reply .codeclimate.yml]: sure

**Review by haxxmaxx — APPROVED** [review]

**Pull request merged**